plugins.txt

```diff
@@ -31,14 +31,14 @@ cloudbees-jenkins-advisor:3.3.1
 command-launcher:1.6
 config-file-provider:3.8.0
 configuration-as-code:1.51
-credentials:2.4.1
-credentials-binding:1.24
+credentials:2.5
+credentials-binding:1.25
```

Comment:
Only features and documentation: https://github.com/jenkinsci/credentials-binding-plugin/releases/tag/credentials-binding-1.25